js/moveset_import.js

```diff
@@ -262,7 +260,11 @@ function addSets(pokes) {
 					currentPoke.nameProp = "Custom Set";
 				}
 				currentPoke.isCustomSet = true;
-				currentPoke.ability = getAbility(rows[i + 1].split(":"));
+				if (currentPoke.name.indexOf("-Mega") !== -1) {
+					currentPoke.ability = getAbility(rows[i + 1].split(":"));
+				} else {
```

Comment:
will probably need something like ``currentPoke.name.indexOf("-Mega-X) {currentPoke.ability = POKEDEX_SM[currentPoke.name].ability;}`` and ``currentPoke.name.indexOf("-Mega-Y") {currentPoke.ability = POKEDEX_SM[currentPoke.name].altAbility;``